config/defaults.conf

```diff
@@ -70,8 +70,11 @@ do_full_eval = 1     // Evaluate the model on the tasks on target_tasks.
 
 // Related configuration
 load_model = 1  // If true, restore from checkpoint when starting do_pretrain. No impact on do_target_task_training.
-transfer_paradigm = "frozen" // How to do learning on top of the pretrained model
+transfer_paradigm = "frozen" // How to do learning on top of the task-independently pretrained model
+                             // Note: this option only works for models like ELMO/GPT/BERT (models that was pretrained on large amount of data)
```

Comment:
why wouldn't these apply if we're using LSTMs or some non pre-trained models?

Reply:
you are right, I don't what I was thinking.